### PR TITLE
Simplify setting up debugging

### PR DESCRIPTION
**Pull request by jamiebuilds**
re: https://github.com/jamiebuilds/unstated/commit/70be71d5064c8cdfc1898a56589cdbd3ffede83a

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,82 +1,66 @@
 'use strict';
 const {detailedDiff} = require('deep-object-diff');
-
-module.exports = options => {
-	options = {
-		isEnabled: true,
-		logStateChanges: true,
-		...options
```
Comment (damassi) [inline index.js:8]: Just a note that this errors out in create react app: 

screen shot 2018-04-12 at 4 57 38 pm

File: readme.md [hunk 2]
```diff
@@ -15,53 +15,26 @@ $ npm install unstated-debug
...
-	// …
-}
+import React from 'react';
+import { render } from 'react-dom';
+import { Provide } from 'unstated';
+import App from './components/App';
+import UNSTATED from 'unstated-debug';
+
+UNSTATED.isEnabled = true;
+UNSTATED.logStateChanges = true;
```
Comment (damassi) [inline readme.md:28]: Why go this route rather than treating this like middleware? This approach seems much less flexible.

File: index.js (hunk 1 above)
Comment (jamiebuilds) [reply index.js:8]: that is deleted code

File: readme.md (hunk 2 above)
Comment (jamiebuilds) [reply readme.md:28]: Cause middleware is stupid and people do stupid shit with it

File: index.js (hunk 1 above)
Comment (damassi) [reply index.js:8]: If this PR for whatever reason doesn't get merged I wanted to point this out for the library author

File: readme.md (hunk 2 above)
Comment (damassi) [reply readme.md:28]: charming as ever 🎉

Comment (jamiebuilds) [reply readme.md:28]: 💋

**Comment by sindresorhus** [comment]
Looks and works great. This is so much nicer. Thanks for adding the required hook and this PR, @jamiebuilds :)

**Pull request merged**